BMM.Core/Translation/en/main.json

```diff
@@ -632,5 +632,10 @@
     "Transcription": "Transcription",
     "Melody": "Melody",
     "Text": "Text"
+  },
+  "HvheDetailsViewModel": {
+    "Title": "Global Score",
+    "Boys": "Boys",
```

Comment:
the response includes the translated text for Boys & Girls. Can we use that?

Reply:
fixed